float64 time
ActuatorState[] actuator_states
JointState[] joint_states
